test/integration/server/add_test.go

```diff
@@ -86,36 +86,40 @@ var _ = Describe("AddApplication", func() {
 			actual, err := helpers.GetFilesForPullRequest(ctx, gh, org, sourceRepoName, fs)
 			Expect(err).NotTo(HaveOccurred())
 
-			expectedKustomization := kustomizev2.KustomizationSpec{
+			expectedKustomization := kustomizev1.KustomizationSpec{
 				// Flux adds a prepending `./` to path arguments that doen't already have it.
 				// https://github.com/fluxcd/flux2/blob/ca496d393d993ac5119ed84f83e010b8fe918c53/cmd/flux/create_kustomization.go#L115
 				Path: "./" + req.Path,
 				// Flux kustomization default; I couldn't find an export default from the package.
 				Interval: metav1.Duration{Duration: time.Duration(1 * time.Minute)},
 				Prune:    true,
-				SourceRef: kustomizev2.CrossNamespaceSourceReference{
+				SourceRef: kustomizev1.CrossNamespaceSourceReference{
 					Name: req.Name,
 					Kind: sourcev1.GitRepositoryKind,
 				},
 				Force: false,
 			}
 
+			repoURL, err := gitproviders.NewRepoURL(sourceRepoURL)
+			Expect(err).NotTo(HaveOccurred())
+
 			expectedSource := sourcev1.GitRepositorySpec{
 				URL: req.Url,
 				SecretRef: &meta.LocalObjectReference{
-					Name: app.CreateRepoSecretName(clusterName, sourceRepoURL).String(),
+					Name: automation.CreateRepoSecretName(clusterName, repoURL).String(),
 				},
 				Interval: metav1.Duration{Duration: time.Duration(30 * time.Second)},
 				Reference: &sourcev1.GitRepositoryRef{
 					Branch: req.Branch,
 				},
+				Ignore: helpers.GetIgnoreSpec(),
 			}
 
 			expectedApp := wego.ApplicationSpec{
 				URL:            req.Url,
 				Branch:         req.Branch,
 				Path:           req.Path,
-				ConfigURL:      "",
+				ConfigURL:      req.Url,
```

Comment:
Question: so we automatically populate this for the user if they don't specify a `ConfigURL`, correct?

Reply:
Yes. That happens in `app.Add` when we validate / update the parameters.